### PR TITLE
Polishes copy for onboarding modal

### PR DESCRIPTION
**Pull request by s-ben**
Was checking out the latest politeiagui and couldn't resist polishing some copy on the onboarding modal. Mainly just rewriting for flow. Feel free to close if changes are off message or review would slow down launch.

### CODE REVIEW

File: src/components/Modal/contents/OnBoard.js [hunk 1]
```diff
@@ -86,28 +86,23 @@ const onBoardingContent = [
     body: `
 **What is Politeia?**
 
-Politeia is a platform which is being developed to support Decred’s governance.
-It facilitates the submission, tracking, and discussion of Decred governance proposals by the stakeholder community.
+Politeia is a platform that supports Decred’s governance model. It facilitates the submission, tracking, and discussion of Decred governance proposals by the stakeholder community.
```
Comment (RichardRed0x) [inline src/components/Modal/contents/OnBoard.js]: Don't agree with adding "model", makes it sound like it's supporting some concept.

File: src/components/Modal/contents/OnBoard.js [hunk 2]
```diff
@@ -86,28 +86,23 @@ const onBoardingContent = [
...
+
+Decred’s approach to decision-making is grounded in the principle of stakeholder voting. To vote, stakeholders must buy “tickets” that lock their DCR until they are called to vote. Stakeholders can vote on any open proposal.
```
Comment (RichardRed0x) [inline src/components/Modal/contents/OnBoard.js]: tickets "that lock their DCR" doesn't sound right, the ticket doesn't lock anything, the locking is done to obtain the ticket.

Comment (RichardRed0x) [reply src/components/Modal/contents/OnBoard.js]: Would be better to get rid of "buy" altogether.

> To vote, stakeholders must time-lock DCR to buy tickets. While tickets are live they can be used to cast a vote for or against each open Politeia proposal.

**Comment by s-ben** [comment]
@RichardRed0x , agree with both of your suggested changes. Have implemented and pushed to branch.

**Pull request merged**